### PR TITLE
Fixed vulnerabilities

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-4234

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -375,7 +376,8 @@ dependencies {
   annotationProcessor group: 'org.projectlombok', name: 'lombok', version: versions.lombok
 
   implementation group: 'org.projectlombok', name: 'lombok', version: versions.lombok
-  implementation "ch.qos.logback:logback-classic:1.2.6"
+  implementation group: 'ch.qos.logback', name: 'logback-classic', version: versions.logbackVersion
+  implementation group: 'ch.qos.logback', name: 'logback-core', version: versions.logbackVersion
```
Comment (mdayican) [inline build.gradle:380]: why do we need core?

Comment (apereverzin) [reply build.gradle:380]: Otherwise dependency check fails

**Review by matt-k1998 — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (mdayican) [reply build.gradle:380]: why it was working without it?

Comment (apereverzin) [reply build.gradle:380]: be cause of the suppression of the vulnerability CVE-2021-42550

Comment (mdayican) [reply build.gradle:380]: it can not be the main reason. I guess it brings another lib an overrides other one

**Pull request merged**